src/index.html

```diff
@@ -16,17 +16,28 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page__body">
+    <form
+      action="#"
+      method="post"
+    >
+      <div class="search search--big">
```

Comment:
You can do it without this additional div wrapper, so please remove it and move background to some classes that you use for input